src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -318,6 +320,99 @@ type_to_insert_op (MonoType *type)
 	}
 }
 
+struct IntrinGroup {
```

Comment:
We usually add typedefs for structure types to avoid having to write 'struct' everywhere.